about.html

```diff
@@ -0,0 +1,89 @@
+          <a href="#" class="textWhite fs12 decorNone" target="_blank" rel="noreferrer noopener">My Page</a>
+          <a class="textWhite fs12 decorNone" href="https://processing.org/" target="_blank" rel="noreferrer noopener">Logout</a>
+        </div>
+        <div id="popMenu" class="noneMobileToFlex navDeskMenu paddingX5">
+          <a href="index.html" class="logo hAuto noneMobileToBlock"><img class="w100"src="assets/logo.svg" alt="logo"></a>
+
+          <ul class="dFlex textDark fs14">
+            <li id="closeMenuSymbol" class="noneDesktop"><a class="decorNone" href="#">X</a></li>
+            <li><a class="decorNone noneDesktop" href="index.html">Home</a></li>
+            <li><a class="decorNone" href="about.html">About</a></li>
+            <li><a class="decorNone" href="index.html#speakers">Speakers</a></li>
+            <li><a class="decorNone" href="index.html#partners">Partners</a></li>
+            <li><a class="decorNone" href="#">Join</a></li>
+            <li><a class="decorNone" href="#">News</a></li>
+            <li class="orangeColor register"><a class="decorNone" href="#">Register</a></li>
+
+          </ul>
+        </div>
+
+      </nav>
+      <section id="landing" class="dFlex paddingX5 bgImgAbout">
+        <h1 class="orangeColor bigTitle fw800 textAlignCenter">Processing Community Day 2020</h1>
+        <div class="invitationContainer">
+          <p class="textPaagraph textDark textAlignCenter fs14">Processing Community Day will take place on Thursday, October 10, 2020, for which we will have national and international guests who work in the field of Art, Design and New Media.</p>
+          <p class="textPaagraph textDark textAlignCenter fs14">During this day, the guests will give talks and master workshops in which they will share their projects and experience with the use of free software and the application of these tools in various fields.</p>
+        </div>
+          <p class="textDark fs12 textAlignCenter fw400 w75 marginXAuto">Please contact us per Email for any further questions about Processin Community Day 2020</p>
+          <p class="textAlignCenter textDark fs12 fw700"><a href="#">pcd2020@pdcquito.com</a></p>
+      </section>
+    </div>
+    <section id="foundation" class="padSections paddingX5">
+      <h2 class="textDark fs18 fw700 textAlignCenter">About Processing Foundation</h2>
+      <div class="orangeColor guideBar marginXAuto">
+      </div>
+      <p class="textDark fs12 textAlignCenter fw400 w75 marginXAuto">Our mission is to
+        promote software literacy within the visual arts, and visual literacy within
+        technology-related fields — and to make these fields accessible to diverse communities.
+        Our goal is to empower people of all interests and backgrounds to learn how to program
+        and make creative work with code, especially those who might not otherwise have
+        access to these tools and resources.
+      </p>
+    </section>
```

Comment:
- [ ] [OPTIONAL] Great job 👏 You've made great progress. In addition to what you have done, I think it would be better **if you remove the scroll property on this landing section on the about page**. Please take a look at the screenshot below 👇 
![image](https://user-images.githubusercontent.com/90222110/190208364-3c44dc3c-29dd-4181-95e1-42d960ed53eb.png)